packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -155,6 +155,34 @@ module.exports = function (log, config, oauthdb) {
     return time.format('L');
   }
 
+  // Borrowed from fxa-payments-server/src/lib/formats.ts
+  // TODO: Would be nice to share this if/when TypeScript conversion reaches here.
+  const baseCurrencyOptions = {
+    style: 'currency',
+    currencyDisplay: 'symbol',
+  };
+
+  /**
+   * This returns a string that is formatted according to the given locale.
+   *
+   * Borrowed from fxa-payments-server/src/lib/formats.ts
+   * TODO: Would be nice to share this if/when TypeScript conversion reaches here.
+   *
+   * @param {number} amountInCents
+   * @param {string} currency
+   * @param {string} locale
+   */
+  function getLocalizedCurrencyString(
```

Comment:
Oh, and one annoying point of interest: I *wanted* to perform the currency formatting in the Stripe helper. But, the user's account (and thus locale) is not available when data is extracted from webhook events. In fact, that data is used to discover the user's account via email address.

So, rather than try to rework webhook handling, it seemed easiest to push the currency formatting into the email handlers. Other localization logic lives in there anyway.

Reply:
👍